autokeras/blocks/basic.py

```diff
@@ -763,11 +763,18 @@ class XceptionBlock(KerasApplicationBlock):
             If left unspecified, it will be tuned automatically.
     """
 
-    def __init__(self, pretrained: Optional[bool] = None, **kwargs):
+    def __init__(self, pretrained: Optional[bool] = None,
+     min_size: Optional[Union[int, hyperparameters.Choice]] = None, 
+     **kwargs):
         super().__init__(
             pretrained=pretrained,
             models={"xception": applications.Xception},
-            min_size=71,
+            min_size = utils.get_hyperparameter(
+            min_size, 
+            hyperparameters.Choice(
```

Comment:
test failed due to missing `ordered` argument. 
https://github.com/keras-team/autokeras/pull/1542/checks?check_run_id=2248768347#step:7:310